drivers/can/can_mcux_flexcan.c

```diff
@@ -1274,7 +1274,7 @@ static int mcux_flexcan_init(const struct device *dev)
 #ifdef CONFIG_CAN_MCUX_FLEXCAN_FD
 	if (config->flexcan_fd) {
 		flexcan_config.timingConfig.frJumpwidth = data->timing_data.sjw - 1U;
-		flexcan_config.timingConfig.fpropSeg = data->timing_data.prop_seg;
+		flexcan_config.timingConfig.fpropSeg = data->timing_data.prop_seg - 1U;
```

Comment:
Are you sure about this? According to the data sheet, the Fast Propagation Segment field does not specify a value one less than the desired (unlike the other timing fields).

![image](https://github.com/zephyrproject-rtos/zephyr/assets/1076226/7d06a64c-3ef9-4d30-95c0-6bba8f10e76b)

Reply:
I missed that, I assumed every field had a `+ 1`. I will undo this change.